src/js/cf-expandables.js

```diff
@@ -5,110 +5,238 @@
-        $cueOpen.css( 'display', 'inline' );
-        $cueClose.css( 'display', 'none' );
-        $content.attr( 'aria-expanded', 'false' );
-        $target.attr( 'aria-pressed', 'false' );
-        if ( typeof duration === 'undefined' ) {
-          duration = $.fn.expandable.calculateCollapseDuration( $content.height() );
-        }
-        $this.removeClass('expandable__expanded');
-        $content.slideUp({
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
-
-      this.init();
-
-    });
+        this.expand();
+      }
+    },
+
+    /**
+     * Expands the expandable, sets the internal state
+     * and adds aria attributes.
+     *
+     * @param {integer} duration -
+     *   The time duration in which the expansion will occur.
+     */
+    expand: function expand( duration ) {
+      var $content = this.$content;
+      this.$target.attr( 'aria-pressed', 'true' );
+      $content.attr( 'aria-expanded', 'true' );
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+      this.$el.addClass( this.expandedClass );
+      $content.slideDown( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+      this.isExpanded = true;
+    },
+
+    /**
+     * Collapses the expandable, sets the internal state
+     * and adds aria attributes.
+     *
+     * @param {integer} duration -
+     *   The time duration in which the collapse will occur.
+     */
+    collapse: function collapse( duration ) {
+      var $content = this.$content;
+      this.$target.attr( 'aria-pressed', 'false' );
+      $content.attr( 'aria-expanded', 'false' );
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+      this.$el.removeClass( this.expandedClass );
+      $content.slideUp( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+      this.isExpanded = false;
+    }
+  };
 
+  // Expandable Plugin declarations.
+  /**
+   * Instantiates the Expandable and configures
+   * the expandable options.
+   *
+   * @param {object} options - Object used to customize the Expandable.
+   * @returns {object} - jQuery Object.
+   */
+  $.fn.expandable = function expandable( options ) {
+    return this.each( function() {
+      ( options || ( options = {} ) )
+      .$el = $( this );
+      Expandable.create.call( this, options );
+    } );
   };
 
-  $.fn.expandable.calculateExpandDuration = function( height ) {
+  /**
+   * Returns the expand duration based on height parameter
+   * and internal min/max values.
+   *
+   * @param {height} integer - The height of an element.
+   * @returns {integer} - The constrained time duration.
+   */
+  $.fn.expandable.calculateExpandDuration =
+  function calculateExpandDuration( height ) {
     return $.fn.expandable.constrainValue( 450, 900, height * 4 );
```

Comment:
Ah ok, so they just happened to match. I'd agree that it'd be nice if the values here and below were configurable.